src/index.ts

```diff
@@ -0,0 +1,212 @@
+import {
+  getInput,
+  getMultilineInput,
+  info,
+  setFailed,
+  endGroup,
+  startGroup,
+} from "@actions/core";
+import { spawnSync } from "node:child_process";
+import { existsSync } from "node:fs";
+import * as path from "node:path";
+import * as util from "util";
+const exec = util.promisify(require("child_process").exec);
+
+const DEFAULT_WRANGLER_VERSION = "^3.0.0";
```

Comment:
Maybe this should be pinned to the action version?

Reply:
Can do